#
# fop fix to build our documentation on CentOS 6
#
# Inverse inc. <info@inverse.ca>
#
--- /usr/bin/fop.orig   2012-01-17 21:25:50.000000000 -0500
+++ /usr/bin/fop        2012-01-17 21:26:04.000000000 -0500
@@ -26,7 +26,7 @@
 # Set parameters
 set_jvm
 set_classpath commons-io batik-all avalon-framework xmlgraphics-commons \
-  commons-logging fop
+  commons-logging fop xalan-j2
 set_flags $BASE_FLAGS
 set_options $BASE_OPTIONS
 
